superset/views/core.py

```diff
@@ -2587,9 +2588,7 @@ def queries_exec(last_updated_ms: Union[float, int]) -> FlaskResponse:
 
         sql_queries = (
             db.session.query(Query)
-            .filter(
-                Query.user_id == g.user.get_id(), Query.changed_on >= last_updated_dt
-            )
+            .filter(Query.user_id == get_user_id(), Query.changed_on >= last_updated_dt)
```

Comment:
Mixed types, though likely benign from the database's perspective—at least for MySQL.